Automattic/wp-calypso
# Site Dashboard: Removing SitePreviewPaneTabs and replacing with SectionNav and NavTabs

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/52


## Proposed Changes

* This PR removes the SitePreviewPaneTabs and replaces them with SectionNav and NavTabs components. This is to meet design requests.
* This PR also modifies the style of the navigation dropdowns for smaller screens to match design specs.
* This PR includes a fix for mobile displays as well where the sites overview section was dsiplaying when it shouldn't.

Additional Figma file reference for mobile views: J7ZbWnnIYpC2THVlktTM23-fi-4717_218380

## Testing Instructions

For the below testing steps, make sure to run `yarn start-a8c-for-agencies` locally and then visit `http://agencies.localhost:3000/sites`.

* In testing, the site preview pane navigation should match what it was before.
* However there is one change, and that is that on mobile screens the navigation becomes a dropdown. This matches previous design specs, available from the Figma link above.

**Screenshots**

Dropdown navigation in preview pane:

<img width="484" alt="Screenshot 2024-03-19 at 18 25 02" src="https://github.com/Automattic/wp-calypso/assets/16754605/71ee6208-bbcd-482a-80a1-74df0828966f">


## Pre-merge Checklist
